Objects/obmalloc.c

```diff
@@ -467,6 +467,7 @@ _PyMem_ArenaAlloc(void *Py_UNUSED(ctx), size_t size)
     if (ptr == MAP_FAILED)
         return NULL;
     assert(ptr != NULL);
+    _PyAnnotateMemoryMap(ptr, size, "cpython:PyMem_ArenaAlloc");
```

Comment:
obmalloc only uses mmap for arenas
```suggestion
    _PyAnnotateMemoryMap(ptr, size, "cpython:obmalloc");
```